app/workers/queue_listener.rb

```diff
@@ -6,22 +6,18 @@ class QueueListener
   # shoryuken_options queue: 'post_test', auto_delete: true, body_parser: :json
 
   def perform(sqs_message, params)
-    params = params.symbolize_keys
```

Comment:
Rails' `params` is a `HashWithIndifferentAccess`, so you can access it with either type of key.